tsconfig.json

```diff
@@ -2,18 +2,17 @@
   "compilerOptions": {
     "outDir": "./dist",
     "sourceMap": true,
-    "declaration": true,
-    "strictNullChecks": true,
-    "noImplicitAny": false,
+    "strict": true,
+    "jsx": "preserve",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
-    "module": "CommonJS",
-    "target": "ES2016",
+    "module": "esnext",
```

Comment:
Targeting ES6 is probably fine. Commonjs is necessary for typescript-demo-lib because they need to produce libraries for node-consumption. @robert-cronin see this.

Reply:
Changed the module back to ComonJS.